frameworks/javascript/cw-2.js

```diff
@@ -108,9 +119,7 @@ try {
     },
 
     // backwards compatibility
-    format: function(obj, options) {
-      Test.display.format(obj, options);
-    },
+    format: Test.display.format,
```

Comment:
This doesn't work because `Test.display` is still `undefined` at this point.
Please test before opening PR and pushing changes.